docs/changelog.rst

```diff
@@ -2,6 +2,13 @@ CbAPI Changelog
 ===============
 .. top-of-changelog (DO NOT REMOVE THIS COMMENT)
 
+CbAPI 1.6.2 - Released April 08, 2020
+-------------------------------------
+
+Updates
+
+.. (add your updates here)
```

Comment:
This needs updated

Reply:
Yep, currently waiting on the finalized text!